docs/operations/data-center/index.md

```diff
@@ -0,0 +1,72 @@
+
+## Step-by-step
+
+### Setup your Environment
+
+--8<-- "docs/shared/ccloud_setup.md"
+
+### Read the data in
+
+Our data center power analysis applications require data from two different sources: customer tenant information and smart control panel readings.
+
+Typically, customer information would be sourced from an existing database. As customer occupancy changes, tables in the database would be updated and we can capture those changes and stream them into Kafka with Kafka Connect using [Change Data Capture](https://www.confluent.io/blog/cdc-and-streaming-analytics-using-debezium-kafka/).
+
+Telemetry data may be sourced into Kafka in a variety of ways. MQTT is a popular source for Internet of Things (IoT) devices, and smart electrical panels may provide this functionality out of the box. If the panel data can be sourced from MQTT, an [MQTT Connector](https://docs.confluent.io/kafka-connect-mqtt/current/mqtt-source-connector/index.html) is available to bridge MQTT and Kafka.
+
+Below are sample Kafka Connect configurations you could use to deploy source connectors to read data from their originating systems. 
+
+```sql
+--8<-- "docs/operations/data-center/source.json"
+```
+
+The above `MySqlCdcSource` configuration could be used to stream changes from the `customer` database's `tenant` table into the Kafka cluster. This connector is provided as [fully managed by Confluent Cloud](https://docs.confluent.io/cloud/current/connectors/cc-mysql-source-cdc-debezium.html).  And the same is true for the telemtry data, which can be sourced using the [fully managed MQTT source Connector](https://docs.confluent.io/cloud/current/connectors/cc-mqtt-source.html) on Confluent Cloud.
+
+Fully managed connectors can be deployed using the web console or the Confluent CLI command `confluent connect create --config <file>`.
+
+If you do not have equivalent source systems to configure connectors for, you can still proceed with this recipe and utilize the sample `INSERT` statements provided below to simulate the tenant occupancy and telemetry data.
+
+### Run stream processing app
+
+Now you can process the data in a variety of ways.
+
+```sql
+--8<-- "docs/operations/data-center/process.sql"
+```
+
+The current state of customer tenant occupancy can be represented with a ksqlDB `TABLE`. Events streamed into the `tenant-occupancy` topic represent a customer (`customer_id`) beginning an occupancy of a particular tenant (`tenant_id`). As events are observed on the `tenant-occupancy` topic, the table will model the current set of tenant occupants. You can query this table at points in time to determine which customer occupies which tenant. When customers leave a tenant, the source system will need to send a _Tombstone Record_ (an event with a valid `tenant_id` key and a `null` value). ksqlDB will process the tombstone by removing the row with the given key from the table.
```

Comment:
All the description should be moved to a separate section at the end of the recipe in a separate section `##Explanation` (e.g. https://github.com/confluentinc/ksqldb-recipes/pull/17/files)

Reply:
@ybyzek please see latest on this